main.py

```diff
@@ -81,16 +79,31 @@
     # perform splitting
     X_train, y_train, X_test, y_test = ttsplit.fSplitDataset(dAllPatches, dAllLabels, dAllPats, cfg['sSplitting'], patchSize, cfg['patchOverlap'], cfg['dSplitval'], '')
 
+    if patchSize[0] != 40:
```

Comment:
Why only != 40 ?
Patch size is parametrizable -> what does this function perform? Up- and downscaling of the patches prior to training?
-> If yes, should be done inside the patching routine not in main